softdevice_controller/include/sdc.h

```diff
@@ -99,17 +99,20 @@ extern "C" {
  * @{
  */
 
-/** @brief Auxiliary defines, not to be used outside of this file. */
+/* Dummy defines to avoid red CI. TODO DRGN-17651: Remove */
```

Comment:
Is it intentional that this comment is here?

Reply:
Yes, I'm not able to remove them immediately without breaking CI

Reply:
We have an NCS test which uses these directly, and that test needs an update